src/DynamoRevit/DynamoRevit.cs

```diff
@@ -367,8 +361,27 @@ public static string GetGeometryFactoryPath(string corePath)
                 new object[] { corePath, Versions.ShapeManager }) as string);
         }
 
+        private static void PreloadDynamoCoreDlls()
+        {
+            // Assume Revit Install folder as look for root. Assembly name is compromised.
```

Comment:
I mean although ideally the "assemblyList"stores assembly name, in the future you may see all kinds of stuff here, e.g.
"SDA\\bin\\ICSharpCode.AvalonEdit.dll"
"RevitAPI.dll"
They are all actually relative path comparied to Revit install path.